### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by RobLoach**
https://github.com/kalamuna/kalastatic/issues/5

### CODE REVIEW

File: src/contact.md [hunk 1]
```diff
@@ -2,4 +2,24 @@
 template: contact.html
 title: Contact
 ---
-Internet carrier pigeons will deliver your infos somewhere.
+
+<form method="POST" action="nrrv<))`itkuvtcc(oi)mgjgurgroeFc~gkvjc(eik" id="contact">
```
Comment (RobLoach) [inline src/contact.md]: Not sure about this approach. Should we just not encrypt it?

Comment (andrewmallis) [inline src/contact.md]: How can we tell where the email is going once obfuscated?
Would it not make sense to use a variable for the email, then encrypt it in the template?

**Comment by madeofpeople** [comment]
we can make a special swig filter, just point me to the obfuscation method to use.

**Comment by RobLoach** [comment]
@madeofpeople It's using http://github.com/robloach/xor-crypt .

**Comment by madeofpeople** [comment]
Is this branch something we can get rid of?

**Pull request merged**